### PR TITLE
Note about svn assets/ dir, and generating zips for gh/other purposes

### PR DESCRIPTION
**Pull request by benlk**
## Additions

- Ryan's `release.sh` URL, to match /checklists/releases.md
- Add SCAIP as a plugin using release.sh
- How to update the `assets/` dir of the svn repo, which isn't covered by `release.sh`
- How to generate assets-included zips for GitHub releases and other purposes

## Why

#132, and completionism

### CODE REVIEW

File: projects/wordpress-plugins/release.sh.md [hunk 1]
```diff
@@ -19,11 +20,39 @@ This file provides setup, usage, and configuration instructions for `release.sh`
...
+  - If it asks you to enter the password for your computer's username, press the `[enter]` key to get the username prompt, then enter your `wordpress.org` username and password.
+
+WordPress.org's [How to use Subversion](https://developer.wordpress.org/plugins/wordpress-org/how-to-use-subversion/) article may be of interest, but it's focused on releasing releases and not updating assets.
+
 ### Plugins using `release.sh`
 
 This list may be incomplete.
 
 - The original script: https://github.com/publicmediaplatform/pmp-wordpress/blob/master/release.sh
+- The current script: https://gist.github.com/rnagle/40d84cbd5fef86e3de7781fc31b46d94
```
Comment (aschweigert) [inline projects/wordpress-plugins/release.sh.md]: let's move this to something we control

Comment (benlk) [inline projects/wordpress-plugins/release.sh.md]: https://github.com/INN/docs/issues/133

Should it be in this repo, or another?

Comment (aschweigert) [inline projects/wordpress-plugins/release.sh.md]: seems kind of odd to put it in this repo (and i wish gists for organizations were a thing), maybe we should make a reference/starter repo for plugins, kind of like the largo sample child theme, and put this there?

**Comment by benlk** [comment]
ok to merge?
I just tested the assets portion of this with INN/pym-shortcode.

Asana has a ticket for creating the release.sh/general plugin boilerplate repo

**Comment by aschweigert** [comment]
yes, with the caveat that we need an issue here to update this link when the boilerplate repo is ready to go

**Pull request merged**